assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,28 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
-
+let myFunction = () => {};
 // let anotherFunction = function (param) {
 //   return param;
 // };
-
+let anotherFunction = (param) => param;
```

Comment:
You're able to drop the parentheses if you have exactly one parameter (and aren't doing anything too fancy):
```javascript
 let anotherFunction = param => param; 
```